Sources/Styleguide/PointFreeBaseStyles.swift

```diff
@@ -164,40 +167,53 @@ extension Class.pf {
           | Class.border.none
           | Class.type.textDecorationNone
       case .underline:
-        borderStyles = Class.border.none
+        borderStyles = baseUnderlineButtonClass
+          | Class.border.none
           | Class.type.underline
       }
 
       let colorStyles: CssSelector
       switch (style, color) {
       case (.normal, .black):
-        colorStyles = Class.pf.colors.link.white
+        colorStyles =
+          Class.pf.colors.link.white
           | Class.pf.colors.fg.white
```

Comment:
should these all be reindented quick?

Reply:
this is the indentation Xcode gives! not ideal